news/4981.internal

```diff
@@ -0,0 +1 @@
+Upgrade to Cypress 12.17.1 (latest) @sneridagh
```

Comment:
@sneridagh I think we also need an entry in the news folder for the volto-testing package